release-notes/8.0/supported-os.json

```diff
@@ -0,0 +1,345 @@
+                        "Arm32",
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "14",
+                        "13",
+                        "12.1",
+                        "12"
+                    ],
+                    "unsupported-versions": [
+                        "11"
+                    ],                    
+                    "notes": [
+                        "API 21 is used as the minimum SDK target."
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "Apple",
+            "distributions": [
+                {
+                    "id": "ios",
+                    "name": "iOS",
+                    "link": "https://developer.apple.com/ios/",
+                    "lifecycle": "https://support.apple.com/ios/",
+                    "architectures": [
+                        "Arm64"
+                    ],
+                    "supported-versions": [
+                        "17",
+                        "16",
+                        "15"
+                    ],
+                    "notes": [
+                        "iOS 12.2 is used as the minimum SDK target."
+                    ]
+                },
+                {
+                    "id": "ipados",
+                    "name": "iPadOS",
+                    "link": "https://developer.apple.com/ipados/",
+                    "lifecycle": "https://support.apple.com/ipados/",
+                    "architectures": [
+                        "Arm64"
+                    ],
+                    "supported-versions": [
+                        "17",
+                        "16",
+                        "15"
+                    ]
+                },                
+                {
+                    "id": "macos",
+                    "name": "macOS",
+                    "link": "https://developer.apple.com/macos/",
+                    "lifecycle": "https://support.apple.com/macos/",
+                    "architectures": [
+                        "Arm64",
+                        "x64"
+                    ],
+                    "supported-versions": [
+                        "14",
+                        "13",
+                        "12"
+                    ],
+                    "notes" : [
+                        "The iOS and tvOS simulators are supported on macOS Arm64 and x64.",
+                        "The x64 emulator (Rosetta 2) is supported on macOS Arm64.",
+                        "Mac Catalyst apps are supported on macOS Arm64 and x64."
+                    ]
+                },
+                {
+                    "id": "tvos",
+                    "name": "tvOS",
+                    "link": "https://developer.apple.com/tvos/",
+                    "lifecycle": "https://support.apple.com/apple-tv/",
+                    "architectures": [
+                        "Arm64",
```

Comment:
According to https://github.com/dotnet/core/blob/v8.0.0/release-notes/8.0/supported-os.md tvOS is only supported on `Arm64`. `x64` support is only valid for the simulator (already in the notes for `macOS`).

Reply:
Fixed